Library/Formula/curl.rb

```diff
@@ -81,7 +82,7 @@ def install
     args << (build.with?("libssh2") ? "--with-libssh2" : "--without-libssh2")
     args << (build.with?("libidn") ? "--with-libidn" : "--without-libidn")
     args << (build.with?("libmetalink") ? "--with-libmetalink" : "--without-libmetalink")
-    args << (build.with?("gssapi") ? "--with-gssapi" : "--without-gssapi")
+    args << (build.with?("gssapi") ? "--with-gssapi" + (OS.mac? ? "" : "=#{Formula["homebrew/dupes/krb5"].opt_prefix}") : "--without-gssapi")
```

Comment:
> Update 2:
I was able to successfully build curl with GSSAPI support without the change introduced on line 85.
If someone desires to use system's gssapi, then change on line 85 has to go. But I lean towards keeping that change.